[luci-interpreter] release build error

```
06:27:02  /home/jenkins_slave/workspace/nnfw/master/push-nnpackage-test/compiler/luci-interpreter/src/kernels/If.cpp: In member function 'virtual void luci_interpreter::kernels::If::configure()':
06:27:02  /home/jenkins_slave/workspace/nnfw/master/push-nnpackage-test/compiler/luci-interpreter/src/kernels/If.cpp:38:22: error: unused variable 'graph' [-Werror=unused-variable]
06:27:02     for (RuntimeGraph *graph : {_then_graph, _else_graph})
06:27:02                        ^~~~~
06:27:02  cc1plus: all warnings being treated as errors
```

from https://github.com/Samsung/ONE/issues/2902#issuecomment-657802855
and also https://github.com/Samsung/ONE/issues/457#issuecomment-657805585

--- a/compiler/luci-interpreter/src/kernels/If.cpp
+++ b/compiler/luci-interpreter/src/kernels/If.cpp
@@ -37,6 +37,7 @@ void If::configure()
 
   for (RuntimeGraph *graph : {_then_graph, _else_graph})
   {
+    (void)graph;
     assert(graph->getInputTensors().size() == _inputs.size());
     assert(graph->getOutputTensors().size() == _outputs.size());
   }

